Your branch is broken for me (Ubuntu 18.04)

```
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/lib/ccache/cc
-- Check for working C compiler: /usr/lib/ccache/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/lib/ccache/c++
-- Check for working CXX compiler: /usr/lib/ccache/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
CMake Error at CMakeLists.txt:70 (install):
  install TARGETS given no RUNTIME DESTINATION for executable target
  "plater".


-- Configuring incomplete, errors occurred!
See also "/tmp/Plater/plater/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/Plater/plater/build/CMakeFiles/CMakeError.log".
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,6 @@ cmake_minimum_required(VERSION 3.6)
 project(Plater
   VERSION      1.1
   DESCRIPTION  "3D-printer parts placer and plate generator"
-  HOMEPAGE_URL "https://github.com/Rhoban/Plater"
   LANGUAGES    CXX
 )
 
@@ -18,11 +17,11 @@ add_subdirectory(gui)
 # Packaging
 include(GNUInstallDirs)
 set(CMAKE_INSTALL_DOCDIR ${CMAKE_INSTALL_DATAROOTDIR}/doc/plater)
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE TYPE DOC)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git/index)
   find_package(Git REQUIRED)
   execute_process(
-    COMMAND           "${GIT_EXECUTABLE}" describe --long --tags
+    COMMAND           "${GIT_EXECUTABLE}" describe --always --long --tags
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     OUTPUT_VARIABLE   GIT_VERSION
     OUTPUT_STRIP_TRAILING_WHITESPACE

--- a/gui/CMakeLists.txt
+++ b/gui/CMakeLists.txt
@@ -121,7 +121,7 @@ if(WIN32)
   endif()
 endif()
 
-install(TARGETS plater-gui RUNTIME)
+install(TARGETS plater-gui RUNTIME DESTINATION bin)
 if(MINGW)
   # install system DLL dependencies
   foreach(syslib

--- a/plater/CMakeLists.txt
+++ b/plater/CMakeLists.txt
@@ -67,4 +67,4 @@ if(MINGW)
 endif()
 target_link_libraries(plater libplater)
 
-install(TARGETS plater RUNTIME)
+install(TARGETS plater RUNTIME DESTINATION bin)

